Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:7.9.0-alpine
```

Comment:
chewie doesnt work with this version (not sure about this exact one but I had many issues with >6.0.
4.2.1 would be more reliable